src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -172,22 +171,15 @@ protected void ProcessAssemblyAction (AssemblyDefinition assembly)
 				break;
 
 			case AssemblyAction.CopyUsed:
+				if (!Context.KeepTypeForwarderOnlyAssemblies && SweepTypeForwarders (assembly)) {
```

Comment:
Is the `KeepTypeForwarderOnlyAssemblies` condition here is to somewhat preserve the behavior that we had here in MarkStep?
https://github.com/mono/linker/blob/252c6652221834051f07b92fc9733e42a755ee36/src/linker/Linker.Steps/MarkStep.cs#L366

What goes wrong if we just remove this part of the condition? I thought that `KeepTypeForwarderOnlyAssemblies` would still allow sweeping forwarders, just not removing the assembly.